admin-dev/themes/new-theme/.webpack/common.js

```diff
@@ -186,5 +187,8 @@ module.exports = {
       $: 'jquery', // needed for jquery-ui
       jQuery: 'jquery',
     }),
+    new CopyPlugin([
+      { from: 'static' },
```

Comment:
Is it me or this directory doesn't exist and you just moved everything to public?

Reply:
Oh my mistake, you moved AND copied them.